awx/api/views/__init__.py

```diff
@@ -2563,10 +2563,37 @@ def _validate_spec_data(new_spec, old_spec):
                         return Response(dict(error=_(
                             "The {min_or_max} limit in survey question {idx} expected to be integer."
                         ).format(min_or_max=key, **context)))
-            if qtype in ['multiplechoice', 'multiselect'] and 'choices' not in survey_item:
-                return Response(dict(error=_(
-                    "Survey question {idx} of type {survey_item[type]} must specify choices.".format(**context)
-                )))
+            # if it's a multiselect or multiple choice, it must have coices listed
```

Comment:
@AlanCoding I added in some extra validation that wasn't there before, but is this the gist of what you were looking for?